src/test/java/com/artipie/asto/S3StorageTest.java

```diff
@@ -92,25 +96,27 @@ void shouldUploadObjectWhenSave(final AmazonS3 client) throws Exception {
     }
 
     @Test
+    @Timeout(5)
```

Comment:
@olegmoz why do we need timeout here? test doesn't hang anymore using `OneOffPublisher`

Reply:
@Vatavuk see above, test did hang without changes in `S3Storage.value()`